addon/components/bread-crumbs.js

```diff
@@ -28,11 +28,11 @@ export default Component.extend({
   reverse: false,
   classNameBindings: [ 'breadCrumbClass' ],
   hasBlock: computed.bool('template').readOnly(),
-  currentRouteName: computed.readOnly('applicationController.currentRouteName'),
+  currentUrl: computed.readOnly('applicationRoute.router.url'),
 
-  routeHierarchy: computed('currentRouteName', 'reverse', {
+  routeHierarchy: computed('currentUrl', 'reverse', {
     get() {
-      const currentRouteName = getWithDefault(this, 'currentRouteName', false);
+      const currentRouteName = getWithDefault(this, 'applicationRoute.controller.currentRouteName', false);
```

Comment:
Can we create a computed property for the `currentRouteName`, much like the one on L31?